jsapp/js/account/security/email/emailSection.module.scss

```diff
@@ -0,0 +1,60 @@
+@use 'scss/_colors';
+@use 'scss/sizes';
+@use 'scss/libs/_mdl';
+
+.root {
+  margin-top: sizes.$x20;
+  margin-bottom: sizes.$x60;
+  padding-top: sizes.$x14;
+  display: flex;
+  border-top: sizes.$x1 solid;
+  border-color: colors.$kobo-gray-92;
+}
+
+.titleSection {
+  flex: 2;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.title {
+  margin: 0;
+  color: colors.$kobo-gray-40;
+  font-weight: 600;
+}
+
+.bodySection {
+  flex: 4;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  margin-bottom: sizes.$x14;
+}
+
+.optionsSection {
+  flex: 3;
+  text-align: right;
+
+  button {
+    margin-top: sizes.$x12;
+    // FIXME: Why does text-align do nothing?
+    right: -78%;
```

Comment:
Maybe because of flex? You could change `.optionsSection` to `display: flex` and align items with flex properties